### PR TITLE
Popups in Projects section

### PR DESCRIPTION
**Pull request by maov19**
The following features were added to the Projects Section:
✅ When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
✅ When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -109,10 +112,10 @@ <h2 class="grey_words">2015</h2>
...
                 </ul>
-            <button class="white_button">See Project</button>
+            <button id="submitTwo" class="white_button">See Project</button>
             </div>
         </div>
-        <div class="project">
+        <div id="third-item" class="project">
             <img class="project_image" src="images/work3.png" alt="third project">
             <img class="project_image_desktop" src="images/project3_desktop.png" alt="third project">
             <div class="desktop_text">
```
Comment (ghost) [inline index.html:121]: - [x] Kindly try as much as you can to match the original design
     - Position your elements correctly. The language elements should be below the image and not above the image
     - Adjust the `border-radius` , `text-color` and also add the relevant icons to the `see live` and `see source` buttons
     - Kindly ensure that your `pop-up`  fits exactly on the viewport of every screen by giving it a height of `100 vh`
     
image

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @maov19 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (maov19) [reply index.html:121]: Position elements - Fixed 🔧 
Adjust borders, text and icons of buttons - Fixed 🔧 
Pop-up height to 100vh - Fixed 🔧

image

**Comment by maov19** [comment]
Dear @frank1738 , 

The requested changes are done!
Please let me know if there is anything else I should work on.

Thanks.

File: index.html (hunk 1 above)
Comment (mckent05) [reply index.html:121]: - You seem to have introduced a bug as a result of the changes you made. It seems like the image takes a lot of the space so I cant view the description section and the links button. Please fix this 🙏 

Your project:
eergwrg

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @maov19, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**